resources/acsl/booleanalgebra.md

```diff
@@ -32,20 +35,51 @@ It is also used to make more specific search expressions. For example, say you w
 
 <br>
 
+# Truth Tables
+
+Say we were to have an expression such as `X and Y`, where we don't know the values of X and Y. To resolve issues like these, we use truth tables! Understanding how truth tables can be crucial in certain situations, such as in the `Find the Solutions` subsection under `Sample Problems`. So, if you don't quite understand what truth tables are or how to set one up, then please continue reading below.
+
+Here's an example:
+
+| X | Y | X and Y |
+| --- | --- | --- |
+| 0 | 0 | 0 |
+| 1 | 0 | 0 |
+| 0 | 1 | 0 |
+| 1 | 1 | 1 |
+
+So, let's break down how exactly this truth table was drawn:
+
+First, a column was made for each of the variable terms. Under these columns, I listed out all possible combinations of values for X and Y. X and Y can both either be 0 or 1; together, they could form the pairs: `(0, 0)`, `(1, 0)`, `(0, 1)`, and `(1, 1)`. Each of these pairs should make up one row in the table.
+
+Then, a column was made for the actual expression itself. In this column, I wrote out all of the resultant values for each row. So, for example, in the first row (excluding the header labels), the values for X and Y were both 0. `X and Y` would thus be `0 and 0`, which simplifies down to `0`. Because of this, I put down a `0` for that row. I then did the same for the other rows.
+
+And that's it! One thing to note is that when you get to longer expressions, it may be wise to break it down and use multiple columns. Let's say the expression was `X and Y or X`. This is how it could be set up:
+
+| | | A | |
+| --- | --- | --- | --- |
+| X | Y | X and Y | A or X |
+| 0 | 0 | 0 | 0 |
+| 1 | 0 | 0 | 1 |
+| 0 | 1 | 0 | 0 |
+| 1 | 1 | 1 | 1 |
+
+For this, I broke the expression into two smaller parts. I used `A` to symbolize `X and Y` simply because I didn't want to write out `X and Y or X` in the last column header.
+
+<br>
+
 # Basic Operators
 
 | Operator | How It's Denoted | Description |
 | --- | --- | --- |
 | and | `xy` or `x • y` | The resulting value is only `true` (or 1) if x and y are *both* true. |
```

Comment:
Look up any symbols you don't know here:
https://oeis.org/wiki/List_of_LaTeX_mathematical_symbols